.circleci/build-gulp-assets.sh

```diff
@@ -23,8 +23,28 @@ do
 	cd ${d%/*}
 
 	# Install any dependencies, if we find packages.json
-	[ -f 'package.json' ] && echo -e "\npackage.json found, running 'npm install'"
-	[ -f 'package.json' ] && npm install
+    if [ -f 'package.json' ]
+    then
+
+        echo -e "\npackage.json found in ${d%/*}"
+
+        NODE_SASS_INSTALLED=$(npm list | grep node-sass >/dev/null)
+        if [ -z $NODE_SASS_INSTALLED ]
+        then
+            # this is necessary as I run MacOS locally but Linux for automated builds
```

Comment:
Rebuilding the `node-sass` binary, which is necessary as Lando runs Linux and I run Mac, takes a long time. I am at home with a fast connection. When traveling or with no connection this would be really rough.